fedimint-client/src/lib.rs

```diff
@@ -1513,6 +1506,17 @@ async fn get_config(
     config_res
 }
 
+pub async fn get_config_from_db(db: &Database) -> Option<ClientConfig> {
```

Comment:
The docs for [`with_invite_code`](https://docs.fedimint.org/fedimint_client/struct.ClientBuilder.html#method.with_invite_code) should refer to this function as it can be used to check if a config already exists. 

While multi-federation wallets will typically have a way to tell which federation the user joined or not single-federation wallets might just want to check if there is any config in the DB.